zk/datadog_checks/zk/zk.py

```diff
@@ -374,7 +374,10 @@ def parse_mntr(self, buf):
                     key, tag_name, tag_val, value = m.groups()
                     tags.append('{}:{}'.format(tag_name, tag_val))
                 else:
-                    key, value = line.split()
+                    try:
+                        key, value = line.split()
+                    except Exception as e:
+                        self.log.warning("Unexpected 'mntr' output:%s - Error:%s", line, e)
```

Comment:
```suggestion
                        self.log.debug("Unexpected 'mntr' output: %s - Error:%s", line, str(e))
                        continue
```

If we can't get key,value then we should continue to the loop instead of erroring further down the function. Perhaps lower to debug? Especially since we're not really using this line `zk_peer_state` right now.